EMentors/app/controllers/users/registrations_controller.rb

```diff
@@ -51,9 +51,12 @@ class Users::RegistrationsController < Devise::RegistrationsController
   # end
 
   # The path used after sign up.
-  # def after_sign_up_path_for(resource)
-  #   super(resource)
-  # end
+  def after_sign_up_path_for(resource)
+    # super(resource)
+    # byebug
+    sign_out resource
```

Comment:
override registration#create method for this

Reply:
Sir I have added confirmable to the devise and now it is not creating sessions. Like after sign up action it is redirecting to the sign in page and also flashing the message your email is confirmed.